pegen/pegen.h

```diff
@@ -109,6 +111,12 @@ void *CONSTRUCTOR(Parser *p, ...);
 #define EXTRA start_lineno, start_col_offset, end_lineno, end_col_offset, p->arena
 
 PyObject *new_identifier(Parser *, char *);
+
+static inline void propagate_error(Parser *p) {
+    longjmp(p->error_env, 1); \
```

Comment:
The backslash can probably go now, right?

```suggestion
    longjmp(p->error_env, 1);
```

Reply:
Thanks for the catch! At the end, I decided to inline this and generate it in the c generator.